core/types/transaction_marshalling.go

```diff
@@ -424,7 +432,7 @@ func (tx *Transaction) UnmarshalJSON(input []byte) error {
 			return errors.New("missing required field 'input' in transaction")
 		}
 		itx.Data = *dec.Input
-		itx.Sender = dec.Sender
+		itx.Sender = *dec.Sender
```

Comment:
Need to check that `Sender` is not `nil`

Reply:
fixed in [c6563da](https://github.com/scroll-tech/go-ethereum/pull/630/commits/c6563daee9b272e0f5f419e9d68d089c9991b325)